intellij/src/saros/intellij/editor/annotations/AnnotationHighlighterLayers.java

```diff
@@ -0,0 +1,45 @@
+package saros.intellij.editor.annotations;
+
+import com.intellij.openapi.editor.markup.HighlighterLayer;
+
+/**
+ * Constants defining the different highlighter layers used by Saros editor annotations.
+ *
+ * <p>Annotations on a layer with a higher number overshadow annotations on a layer with a lower
+ * number.
+ *
+ * @see HighlighterLayer
+ */
+class AnnotationHighlighterLayers {
+
+  private AnnotationHighlighterLayers() {}
+
+  /**
+   * Base highlighter layer used for Saros editor annotations. All Saros annotations must be
+   * displayed on or below this highlighter layer.
+   *
+   * @see HighlighterLayer
+   */
+  private static int SAROS_BASE_HIGHLIGHTER_LAYER = HighlighterLayer.CARET_ROW - 1;
+
+  /**
+   * Highlighter layer for the caret annotations displayed as part of the selection annotations.
+   *
+   * @see SelectionAnnotation
+   */
+  static int CARET_HIGHLIGHTER_LAYER = SAROS_BASE_HIGHLIGHTER_LAYER - 1;
```

Comment:
Is there a reason to not make the fields (and class) final ?

Reply:
No, it's supposed to be final. I forgot. Will adjust it right away.

Reply:
Done.